.vscode/extensions.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Can we extract updating the `.vscode` settings into an own commit?

Reply:
Done, added a new commit that adds those configuration files.